kubeutils/annotations.go

```diff
@@ -0,0 +1,29 @@
+package kubeutils
+
+// SoloClusterAnnotation was originally implemented in solo-kitv2's ezkube package
```

Comment:
```suggestion
// SoloClusterAnnotation was originally implemented in skv2's ezkube package
```
(use actual repo name)